Binary files rogerrouter.orig/.git/index and rogerrouter.new/.git/index differ
diff -rU1 rogerrouter.orig/plugins/meson.build rogerrouter.new/plugins/meson.build
--- rogerrouter.orig/plugins/meson.build	2025-01-12 21:05:13.486152903 +0100
+++ rogerrouter.new/plugins/meson.build	2025-01-12 21:12:38.899504028 +0100
@@ -10,3 +10,4 @@
 
-subdir('evolution')
+## 2025-01-12: Fails to build with newev 'evolution-data-server': `/usr/include/evolution-data-server/libedataserver/e-data-server-util.h:263:50: error: unknown type name ‘SoupURIComponent’`. Would work with `evolution-data-server` version 3.44.4 build against libsoup2.4, but it seems not easy to make this build system here specifically using that if it is installed in a special directory, and the up to date `evolution-data-server` installed in main directory. So disable here for convenience of getting this software building.
+# subdir('evolution')
 subdir('gtknotify')
